137 overviews and nav integrity (#309)
FAIRmat-NFDI/nomad-docs: docs/tutorial/overview.md

---
hide: toc
---

# Tutorials

> Tutorials provide guided, learning-oriented paths. Use them when you want to understand a workflow end to end. See {{ nav_link("howto/overview.md", breadcrumb=True) }} if you are trying to complete a known, isolated task.

Reviewer: ```suggestion
Tutorials provide guided, learning-oriented paths. Follow them to learn NOMAD and understand new concepts by working through practical examples. See {{ nav_link("howto/overview.md", breadcrumb=True) }} if you already know your goal and need instructions to accomplish it.
```

> ## Before you begin

Reviewer: I compared the overview pages of all four documentation sections, and I would recommend harmonizing their structure more consistently.

For the Tutorials page, and in line with the proposed PR of creating cleaner and more effective landing pages, I would recommend removing the `Before you begin` section from the overview page. This information is already repeated throughout the relevant tutorial pages, and I think users visiting the overview page primarily want to understand:

- What the Tutorials section is for.
- What learning areas it covers.
- Which related documentation sections or other references they may want to consult.

> ## Before you begin

Reviewer: ```suggestion
```

> Unless stated otherwise, these tutorials use **NOMAD Central**, the public FAIRmat-hosted NOMAD platform. There are two relevant deployments:

Reviewer: ```suggestion
```

> - [**Production**](https://nomad-lab.eu/prod/v1/gui/search/entries){:target="_blank" rel="noopener"}: Use by default for any generic tutorial activities.

Reviewer: ```suggestion
```

> - [**Test**](https://nomad-lab.eu/prod/v1/test/gui/search/entries){:target="_blank" rel="noopener"}: Use primarily for testing publishing capabilities. This deployment contains a dedicated *temporary* database that is routinely wiped.

Reviewer: ```suggestion
```

> The majority of underlying concepts and workflows also apply to **NOMAD Oasis** deployments, although specific behavior may differ between installations depending on which plugins are installed.

Reviewer: ```suggestion
```

> ### Create a NOMAD user account

Reviewer: ```suggestion
```

Reviewer: If we agree on this, I can creat commits to this PR with updated links to the How-to guide in the respective tutorial pages. {{ nav_link("howto/manage/gui/account.md", breadcrumb=True) }}

> Some tutorials require you to sign in before you can upload, share, or publish data. For step-by-step account creation and sign-in guidance, see {{ nav_link("howto/manage/gui/account.md", breadcrumb=True) }}.

Reviewer: ```suggestion
```

## Tutorial paths

<div markdown="block" class="home-grid">
<div markdown="block">

> ### {{ nav_link("tutorial/explore.md") }}

Reviewer: ```suggestion
### Explore published data
```

> Learn how to search, filter, and inspect published NOMAD data.

Reviewer: ```suggestion
Learn how to search, filter, and inspect published NOMAD data.

- {{ nav_link("tutorial/explore.md") }}
```

> Learn how to search, filter, and inspect published NOMAD data.

Reviewer: ```suggestion
Search, filter, and explore data published in NOMAD, and create dashboards using custom widgets.
```

</div>
<div markdown="block">

### Upload and publish data

> Follow a complete upload and publication workflow using either interface.

Reviewer: ```suggestion
Upload, manage, and publish data using the NOMAD GUI or API.
```

- {{ nav_link("tutorial/upload_publish.md") }}
- {{ nav_link("tutorial/upload_publish_api.md") }}

</div>
<div markdown="block">

### Use NOMAD as an ELN

> Create structured experimental records using built-in or custom schemas.

Reviewer: ```suggestion
Create and managed structured experimental records using built-in or custom ELN schemas.
```

- {{ nav_link("tutorial/eln/built_in_templates.md") }}
- {{ nav_link("tutorial/eln/custom_eln_yaml.md") }}
- {{ nav_link("tutorial/eln/tabular_parser_yaml.md") }}

</div>
<div markdown="block">

> ### {{ nav_link("tutorial/workflows_projects.md") }}

Reviewer: ```suggestion
### Manage workflows and projects
```

> Connect related entries and organize them into projects and workflows.

Reviewer: ```suggestion
Connect related entries and organize them into projects and workflows.

- {{ nav_link("tutorial/workflows_projects.md") }}
```

Reviewer: For better consistency, I would change the title of the Manage workflows and projects page to  `Manage workflows and projects in NOMAD` so that it shows in this way in the list.

</div>
<div markdown="block">

### Develop a NOMAD plugin

Build a plugin structure, schema package, and parser through guided exercises.

- {{ nav_link("tutorial/develop_plugin/plugin_structure.md") }}
- {{ nav_link("tutorial/develop_plugin/create_schema_package.md") }}

> - **Create a parser**

Reviewer: ```suggestion
```

>     - {{ nav_link("tutorial/develop_plugin/create_parser.md") }}

Reviewer: ```suggestion
- {{ nav_link("tutorial/develop_plugin/create_parser.md") }}
```

    - {{ nav_link("tutorial/develop_plugin/create_parser_parser_only.md") }}
    - {{ nav_link("tutorial/develop_plugin/create_parser_eln_only.md") }}
    - {{ nav_link("tutorial/develop_plugin/create_parser_hybrid.md") }}

</div>
</div>

> ## Additional training resources

Reviewer: I propose to harmonize this section for all overview pages. 
Right now we have: 
- Additional training resources. 
- When not to use this section
- Related material

How about having `Related materials` as a universal title, with some standard references to the other sections + other relevant resouces?

I will add suggestion to each overview page, to facilitate discussion and decision on this point.

> ## Additional training resources

Reviewer: ```suggestion
## Related material
```

Find event materials and external training collections in the

> {{ nav_link("reference/tutorials.md", breadcrumb=True) }}.

Reviewer: ```suggestion
- Find event materials and external training collections in {{ nav_link("reference/tutorials.md", breadcrumb=True) }}.
- For task-oriented instructions, see {{ nav_link("howto/overview.md", breadcrumb=True) }}.
- For conceptual background, see {{ nav_link("explanation/overview.md", breadcrumb=True) }}.
- For authoritative technical details, see {{ nav_link("reference/overview.md", breadcrumb=True) }}.

```
